src/_locales/pl-PL.json

```diff
@@ -0,0 +1,96 @@
+{
+  "audio-device-communications": "Domyślne urządzenie do komunikacji",
+  "audio-device-default": "Domyślne urządzenie",
+  "audio-device-unknown": "Nieznane urządzenie",
+
+  "button-test-ok": "OK",
+  "button-text-close": "Zamknij",
+  "button-text-dont-tell-me-again": "Nie pokazuj więcej",
+  "button-text-lets-do-this": "Zróbmy to!!",
+  "button-text-lets-go": "Zaczynajmy!!",
+  "button-text-not-now": "Nie teraz",
+
+  "label-about": "Informacje",
+  "label-desktop-settings": "Ustawienia Aplikacji",

```

Comment:
"Ustawienia aplikacji" Capitalize first letter is not required when it isn't context of think with self name (Like name of person or name of product etc). Other application go this way when translate this word. When name of app will occur then is capitalize first letter.